### PR TITLE
list filter config field class

### PR DESCRIPTION
**Pull request by gonzakpo**
- { property: 'status', type_options: { attr: { class: 'col-sm-3' } } }
- { property: 'organization', type_options: { attr: { class: 'col-sm-9' } } }

### CODE REVIEW

**Comment by alterphp** [comment]
Hi @gonzakpo !

Can you post the generated HTML code for the whole list filters `<div>` ? I have a doubt about the class attribute that may affects the input itself (and not only its container).

**Comment by gonzakpo** [comment]
@alterphp 
The value is default col-sm-4, now it allows to configure.

**Comment by alterphp** [comment]
HI @gonzakpo !

- This breaks submit button full height. Why did you change the whole HTML structure ? Only line 91 is useful regarding your intention (modify filter fields covered columns).

- As I expected, the input itself takes the class... (not only the container, the input, select, etc). That's why I asked you to post the generated HTML code...

**Comment by gonzakpo** [comment]
if you use bootstrap <div class = "col"> it goes inside <div class = "row"> always

**Comment by alterphp** [comment]
I just pushed a better bootstrap 3 integration for the list filters form. It keeps full height on submit button.

But still no good solution for your intention.

**Comment by gonzakpo** [comment]
i will see

File: src/Controller/AdminController.php [hunk 1]
```diff
@@ -34,6 +34,7 @@ protected function isActionAllowed($actionName)
         switch ($actionName) {
             // autocomplete action is mapped to list action for access permissions
             case 'autocomplete':
+                break;
```
Comment (alterphp) [inline src/Controller/AdminController.php]: This is not related to the PR, so please don't mix many purposes in the same PR.

And moreover, it is wrong.

**Comment by alterphp** [comment]
Closed as no solution has emerged.

**Comment by gonzakpo** [comment]
right way:
http://php.net/manual/en/control-structures.switch.php

I will create a new pull request
